frontend/src/pages/ActivityReport/Pages/PrintSummary.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Grid,
+} from '@trussworks/react-uswds';
+
+const PrintSummary = ({ reportCreator }) => {
+  const { name, role = '' } = reportCreator;
+
+  return (
+    <div className="font-family-sans smart-hub-meta-summary grid-container print-only">
+      <Grid row>
+        <Grid col={6}>
+          Report Creator
```

Comment:
Adding report id and status would probably be good.

Reply:
Yeah, report status is on the ticket (last checkmark), but I wanted to check if this was being implemented elsewhere since its also part of the web UI.

I could add the report id, but per our meeting, it would be on every header as part of the URL (provided that is enabled when printing). I can check on that.

Reply:
I did end up adding the report Id, but per #162 , there might be a need to use a particular format for display.